Netable/Netable/Request.swift

```diff
@@ -29,6 +29,10 @@ public protocol Request {
     /// Parameters to be encoded and sent with the request.
     var parameters: Parameters { get }
 
+    /// Parameter keys whose values will be printed in full to logs.
+    /// By default, all parameters will be printed as `<REDACTED>` to logs.
+    var safeParameterKeys: Set<String> { get }
```

Comment:
Different term than "safe" might be appropriate here. I think that might be what we call this sort of thing in steamclog, but there is a little more context there ("safe" in a logging framework does sort of imply "safe to log", though now that I think about it, I wouldn't be opposed to changing it in steamclog as well if we can come up with a better name).

Maybe it should just be "unredacted"?

Reply:
is `loggable` or `logged` better?

Reply:
or, if we choose to invert the logic and not redact by default, we could change this to `redactedParameterKeys`?

Reply:
On second though, just going to rename them to `unredactedParameters`

Reply:
I do think having the stuff you list be the ones not to redact is correct, of you want redaction at all, you probably want to redact any given parameter by default, since the biggest privacy risk is problaby a newly added parameter leaking if you need to remember to also change the list of parameters to hide it.